web/docs/writingguide.md

```diff
@@ -148,11 +147,11 @@ This might seem redundant, but it makes our examples future-proof and helps reas
     - "This is actually intentional, but thanks for taking the time to contribute. 😊"
   - **Listen to what people are saying and mirror if you're not sure you're understanding correctly.** This can help validate people's feelings and experiences, while also understanding if _you're_ understanding _them_ correctly.
   - **Use a lot of positive and empathetic emojis.** It's always better to seem a little strange than mean or impatient.
-    This is primarily applicable for Wasp team members speaking to outside contributors. Since most of the core team knows each other pretty well, there's no need to go overboard with the emojis and pleasentries.
+    This is primarily applicable to Wasp team members speaking to outside contributors. Since most of the core team knows each other pretty well, there's no need to go overboard with the emojis and pleasantries.
   - **Kindly communicate rules/boundaries.** If someone behaves in a way that's abusive/inappropriate, respond only with kindness and maturity, but also make it clear that this behavior is not acceptable and what will happen (according to the code of conduct) if they continue behaving poorly.
 - **All docs must go through the review cycle, preferably with more than a single reviewer.** Different people focus on different things. Some of us are great at coming up with examples, others easily come up with analogies and explain complex topics, some have a clear and concise writing style, etc. Therefore, try to get at least two or three people to review your document.
 
 ## Possible improvements
 
 - Some parts of our docs don't follow all the guidelines outlined in this document. There's no need to start fixing all the issues right away, we can slowly improve the docs as we edit them.
-- We've talked about having a git repo with all the example code so it's easier to copy-paste, test, and maintain.
+- We've talked about having a git repo with all the example code in the docs. This should make it easier to copy-paste, test, and maintain code snippets.
```

Comment:
Aha, so all the example snippets in the code, they would all be somewhere as part of actual example project? All right yeah that could be useful, ok got it.